dockerfiles/hpcc/templates/thor.yaml

```diff
@@ -0,0 +1,124 @@
+{{ range $thor := $.Values.thor -}}
+{{- $masterPort := $thor.masterport | default 20000 -}}
+{{- $slavePort := $thor.slaveport | default 20100 -}}
+{{- $slaveDeploymentName := printf "%s-slave" .name | quote }}
+{{- $serviceName := printf "%s-svc" .name }}
```

Comment:
| quote for consistency

Reply:
has changed a bit in most recent commit, don't quote any of these vars here anymore, because some of them are used in other strings, so quote them when used.